BUILD.md

```diff
@@ -89,7 +89,8 @@ The `npm run watch` script in action:
   * Windows Only
     * WSL2 or any virtual machine running Ubuntu 20.04 LTS
   * Git
-  * NodeJS 16, npm 7 (we recommend using the Node Version Manager (nvm) if available for your OS)
+  * NodeJS v16.13.1, npm v8.1.2 (we recommend using the Node Version Manager (nvm) if available for your OS)
+    * `nvm use 16.13.1`
```

Comment:
@ruzell22 I would say `nvm install 16.13.1` instead of `nvm use ...` because if you are on a fresh installation of nvm, it won't let you `use` a specific version unless it was `install` 'd first (at least this is how it used to work a couple of years back the last time I used it, please do correct me if they changed the behavior in the meantime though)